google/resource_compute_instance_template.go

```diff
@@ -389,6 +389,37 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 				},
 			},
 
+			"shielded_vm_config": {
```

Comment:
The suggestions for compute instance apply here as well, unless this resource behaves differently in your experience.